demos/bookstore/app/utils/context.ts

```diff
@@ -1,5 +1,5 @@
-import { createStorageKey } from '@remix-run/fetch-router'
-import { getContext } from '@remix-run/async-context-middleware'
+import { createStorageKey } from 'remix'
+import { getContext } from 'remix/platform'
```

Comment:
Something feels off to me about this. I have to know that `getContext` relies on `node:async_hooks` internally so that I know to get it from `remix/platform`. Maybe it's better to just import directly from `remix/async-context-middleware` and ditch the whole idea of `remix/platform` entirely.

Reply:
yeah - I think thats a good call.  The sub-export is just an added layer of indirection when it's still a single dependency.